Adds flavors, foods, drinks

Colin-Tel opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->

Adds most of the flavors, foods, and drinks from Nyano/DeltaV.

Some things have been commented out.

**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->
:cl:
- add: Added most of the DeltaV and Nyano food and drink, along with recipes.

ZeroDayDaemon commented:
Should probably wait on this unless its necessary

DebugOk commented:
> Should probably wait on this unless its necessary

Sure thing, but it helps to stay up to date with Nyano

Colin-Tel committed:
Adds flavors, foods, drinks

Adds most of the flavors, foods, and drinks from Nyano/DeltaV.

Some things have been commented out.
